UPGRADE.md

```diff
@@ -2,6 +2,21 @@
 
 In addition to the README's [Harness Upgrade Instructions], please note these specific version upgrade instructions.
 
+## Upgrading from 1.4.x to 1.5.x
+
+The drupal harness no longer sets /app/config, /app/docroot/modules and /app/docroot/profiles to be web-writable at runtime.
+
+This means that modules cannot be installed via the admin interface unless you are running in the development environment.
+
+If you require the old behaviour, set the following attribute in your workspace.yml:
```

Comment:
could we suggest considering a lighter touch of specifically targeting needed writeable module files?

Reply:
Updated